azure-pipelines.yml

```diff
@@ -7,62 +7,62 @@ pr:
 - maint/*
 
 jobs:
-- template: ci/azure/linux.yml
-  parameters:
-    name: Linux
-    vmImage: ubuntu-16.04
-    matrix:
-      Python35-64bit:
-        python.version: '3.5'
-      Python36-64bit:
-        python.version: '3.6'
-      Python38-64bit:
-        python.version: '3.8'
-      Python37-64bit:
-        python.version: '3.7'
-      Python35-64bit + MIN_DEPS:
-        python.version: '3.5'
-        DEPENDS: "cython==0.29 numpy==1.12.0 scipy==1.0 nibabel==3.0.0 h5py==2.5.0 nose"
-      Python37-64bit + MIN_DEPS:
-        python.version: '3.7'
-        DEPENDS: "cython==0.29 numpy==1.15.0 scipy==1.1 nibabel==3.0.0 h5py==2.8.0"
-      Python37-64bit + OPTIONAL_DEPS + COVERAGE:
-        python.version: '3.7'
-        EXTRA_DEPENDS: "scikit_learn pandas statsmodels==0.9.0 tables scipy==1.1"
-        COVERAGE: "1"
-      Python37-64bit + VIZ + COVERAGE:
-        TEST_WITH_XVFB: "1"
-        COVERAGE: "1"
-        python.version: '3.7'
-        MESA_GL_VERSION_OVERRIDE: '3.3'
-        LIBGL_ALWAYS_INDIRECT: 'y'
-        EXTRA_DEPENDS: "scikit_learn vtk fury scipy==1.1 pandas statsmodels==0.9.0 tables xvfbwrapper"
-      Python37-64bit + SDIST:
-        python.version: '3.7'
-        INSTALL_TYPE: "sdist"
-        EXTRA_DEPENDS: "scipy"
-      Python37-64bit + PIP:
-        python.version: '3.7'
-        INSTALL_TYPE: "pip"
-        DEPENDS: "" # Dependency checking should get all needed dependencies
-      Python37-64bit + WHEEL:
-        python.version: '3.7'
-        INSTALL_TYPE: "wheel"
-        EXTRA_DEPENDS: "scipy"
-      Python37-64bit + Requirements:
-        python.version: '3.7'
-        INSTALL_TYPE: "requirements"
-        DEPENDS: ""
-      CONDA Python37-64bit + OPTIONAL_DEPS:
-        python.version: '3.7'
-        EXTRA_DEPENDS: "scikit-learn pandas statsmodels==0.9.0 pytables scipy==1.1"
-        INSTALL_TYPE: "conda"
-      CONDA Python37-64bit:
-        python.version: '3.7'
-        INSTALL_TYPE: "conda"
-      CONDA Python36-64bit:
-        python.version: '3.6'
-        INSTALL_TYPE: "conda"
+# - template: ci/azure/linux.yml
```

Comment:
Why all these comments here?

Reply:
I use this PR to understand @BramshQamar issue with CI's. All this comments are just to accelerate the build process so they are temporary. I will remove them soon.